addon/components/denali-progress.hbs

```diff
@@ -0,0 +1,10 @@
+{{! Copyright 2021, Verizon Media. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
+<div class="d-progress {{this.hasLoaderClass}} {{this.sizeClass}}"  ...attributes>
+    {{#unless this.hasLoaderClass}}
+        {{#if (has-block)}}
+            {{yield (hash Bar=(component "denali-progress/bar" color=@color shade=@shade))}}
```

Comment:
```suggestion
            {{yield (hash Bar=(component "denali-progress/bar" color=this.color shade=this.shade))}}
```

Reply:
I suggested to remove the `@arg` class properties since they're being passed directly to the bar component which will do the check again

Reply:
Ah, I thought of it as more of a documentation win.